swingset/src/main/java/com/nqadmin/swingset/SSComboBox.java

```diff
@@ -76,9 +76,9 @@
  * are the correct techniques.
  * There are a variety of method to change the current combobox item
  * such as
- * {@link #setSelectedMapping(java.lang.Object) setSelectedMapping(Integer)}.
```

Comment:
I originally had `{@link \#setSelectedMapping(Integer)}` but got complaints because it's generic and it wanted to put Object in there. So I rewrote it in two pieces so that `setSelectedMapping(Integer)` is what gets displayed. In the other file, SSDBComboBox, you'll see `setSelectedMapping(Long)`

I don't remember if the complaints came from running maven or the IDE. I'm thinking it must be the IDE, sigh. You're IDE gets it right.